attest/api/proto/attest.proto

```diff
@@ -65,3 +65,62 @@ message NonceMessage {
         /// The explicit nonce.
         fixed64 nonce = 4;
 }
+
```

Comment:
My plan is to move the DcapEvidence and EncalveReportDataContents into this proto file as well so that they're available for the `mc-attest-verifier-types` in the enclave.